integration-tests/test/shared/env.ts

```diff
@@ -73,19 +73,19 @@ export class OptimismEnv {
     if (balance.isZero()) {
       await fundUser(watcher, l1Bridge, utils.parseEther('20'))
     }
-    const l1Messenger = getContractFactory('iOVM_L1CrossDomainMessenger')
+    const l1Messenger = getContractFactory('IL1CrossDomainMessenger')
```

Comment:
Might as well just use `L1CrossDomainMessenger` instead of `IL1CrossDomainMessenger` for consistency.